### PR TITLE
Add comment for flags field

### PR DESCRIPTION
**Pull request by vprithvi**

### CODE REVIEW

File: thrift/jaeger.thrift [hunk 1]
```diff
@@ -58,7 +58,7 @@ struct Span {
   4:  required i64           parentSpanId # since nearly all spans will have parents spans, CHILD_OF refs do not have to be explicit
   5:  required string        operationName
   6:  optional list<SpanRef> references   # causal references to other spans
-  7:  required i32           flags        # tbd
+  7:  required i32           flags        # a bit field used to propagate sampling decisions. 1 signifies a SAMPLED span, 2 signifies a DEBUG span.
```
Comment (black-adder) [inline thrift/jaeger.thrift]: 4 signifies deferred span?

Comment (vprithvi) [inline thrift/jaeger.thrift]: It does, but that PR hasn't been merged yet, I'll add it in when it gets in!

Comment (yurishkuro) [inline thrift/jaeger.thrift]: Rather than doing the comment, why don't we add the constants?

Comment (vprithvi) [inline thrift/jaeger.thrift]: We can, or should we make these enums?

Comment (yurishkuro) [inline thrift/jaeger.thrift]: the field has to remain `i32` to avoid a breaking change, so I don't think enums are useful.

Comment (vprithvi) [inline thrift/jaeger.thrift]: I was thinking that the field remain `i32` but we assign enums to it. I was assuming that thrift enums are `i32` under the hood, but I'll verify.

**Pull request merged**